src/styles/StyleUtils.ts

```diff
@@ -61,48 +65,61 @@ const avatarSizes = {
     [CONST.AVATAR_SIZE.SMALL_NORMAL]: variables.avatarSizeSmallNormal,
 };
 
-const emptyAvatarStyles = {
+const emptyAvatarStyles: Partial<Record<ValueOf<typeof CONST.AVATAR_SIZE>, ViewStyle | CSSProperties>> = {
     [CONST.AVATAR_SIZE.SMALL]: styles.emptyAvatarSmall,
     [CONST.AVATAR_SIZE.MEDIUM]: styles.emptyAvatarMedium,
     [CONST.AVATAR_SIZE.LARGE]: styles.emptyAvatarLarge,
 };
 
+const avatarFontSizes: Partial<Record<ValueOf<typeof CONST.AVATAR_SIZE>, number>> = {
```

Comment:
Can we create a "variable" for `ValueOf<typeof CONST.AVATAR_SIZE>` to follow DRY.

Reply:
Done!